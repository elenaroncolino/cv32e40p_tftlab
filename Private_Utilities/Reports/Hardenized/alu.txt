Version("1.1")
Date("01/20/24 13:01:40")
User("s317644")
Tool("FMGR")
Info("  Type:    Fault Coverage Report")
Info("  Version: T-2022.06-SP2 (64-bit, Nov 15 2022)")
Info("  Models:  All")
Info("  Classes: All")

FunctionalBlocks{
     1 design {
           1  sbst 0ps {F:54240 ND:14053 DD:32 NC:1906 NO:38061 NS:188}
    }
}

FaultInfo{
    FubNum;
    TestNum;
    Attempts;
    NumPot;
}

FaultList{

Total              ND           DD             NC              NO            NS Scope
----- --------------- ------------ -------------- --------------- ------------- -----
54240 14053  (25.91%) 32   (0.06%) 1906   (3.51%) 38061  (70.17%) 188   (0.35%) cv32e40p_top
54240 14053  (25.91%) 32   (0.06%) 1906   (3.51%) 38061  (70.17%) 188   (0.35%) -core_i
54240 14053  (25.91%) 32   (0.06%) 1906   (3.51%) 38061  (70.17%) 188   (0.35%) --ex_stage_i
54240 14053  (25.91%) 32   (0.06%) 1906   (3.51%) 38061  (70.17%) 188   (0.35%) ---alu_i
54240 14053  (25.91%) 32   (0.06%) 1906   (3.51%) 38061  (70.17%) 188   (0.35%) ----alu3 (8 hidden instances, 32 faults)
18100  6379  (35.24%)  0   (0.00%)  951   (5.25%) 10708  (59.16%)  62   (0.34%) -----alu_1 (1399 hidden instances, 11388 faults)
 5940     0   (0.00%)  0   (0.00%)    0   (0.00%)  5884  (99.06%)  56   (0.94%) ------alu_div_i (767 hidden instances, 5940 faults)
  432     0   (0.00%)  0   (0.00%)    0   (0.00%)   432 (100.00%)   0   (0.00%) ------ff_one_i (63 hidden instances, 432 faults)
  340     0   (0.00%)  0   (0.00%)    0   (0.00%)   340 (100.00%)   0   (0.00%) ------popcnt_i (39 hidden instances, 340 faults)
18028  6320  (35.06%)  0   (0.00%)  949   (5.26%) 10695  (59.32%)  64   (0.36%) -----alu_2 (1390 hidden instances, 11332 faults)
 5924     0   (0.00%)  0   (0.00%)    0   (0.00%)  5868  (99.05%)  56   (0.95%) ------alu_div_i (763 hidden instances, 5924 faults)
  432     0   (0.00%)  0   (0.00%)    0   (0.00%)   432 (100.00%)   0   (0.00%) ------ff_one_i (63 hidden instances, 432 faults)
  340     0   (0.00%)  0   (0.00%)    0   (0.00%)   340 (100.00%)   0   (0.00%) ------popcnt_i (39 hidden instances, 340 faults)
18080  1354   (7.49%)  0   (0.00%)    6   (0.03%) 16658  (92.13%)  62   (0.34%) -----alu_3 (1398 hidden instances, 11380 faults)
 5928     0   (0.00%)  0   (0.00%)    0   (0.00%)  5872  (99.06%)  56   (0.94%) ------alu_div_i (764 hidden instances, 5928 faults)
  432     0   (0.00%)  0   (0.00%)    0   (0.00%)   432 (100.00%)   0   (0.00%) ------ff_one_i (63 hidden instances, 432 faults)
  340     0   (0.00%)  0   (0.00%)    0   (0.00%)   340 (100.00%)   0   (0.00%) ------popcnt_i (39 hidden instances, 340 faults)
}
#------------------------------------------------------------------------------
# Faults listed:       Prime: 0        Total: 0
#------------------------------------------------------------------------------
# Fault Coverage Summary
#
#                                        Prime             Total
#------------------------------------------------------------------------------
# Total Faults:                          54240             54240
#
# Dropped Detected                 DD       32   0.06%        32   0.06%
# Dropped Potential                PD        0   0.00%         0   0.00%
# Not Detected                     ND    14053  25.91%     14053  25.91%
# Not Strobed                      NS      188   0.35%       188   0.35%
# Not Observed                     NO    38061  70.17%     38061  70.17%
# Not Controlled                   NC     1906   3.51%      1906   3.51%
#
# Faults not detected yet:
#   Not simulated yet:                   40155  74.03%     40155  74.03%
#   Simulated 1 to 5 times:              14053  25.91%     14053  25.91%
#   Simulated 6 to 10 times:                 0   0.00%         0   0.00%
#   Simulated > 10 times:                    0   0.00%         0   0.00%
#
# Faults potentially detected:
#   Possible 1 to 5 times:                   0   0.00%         0   0.00%
#   Possible 6 to 10 times:                  0   0.00%         0   0.00%
#   Possible > 10 times:                     0   0.00%         0   0.00%
#
# Test Coverage                                  0.06%             0.06%
# Fault Coverage                                 0.06%             0.06%
#------------------------------------------------------------------------------
