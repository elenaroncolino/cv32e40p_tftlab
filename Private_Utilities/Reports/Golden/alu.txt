Version("1.1")
Date("01/19/24 01:39:53")
User("s314703")
Tool("FMGR")
Info("  Type:    Fault Coverage Report")
Info("  Version: T-2022.06-SP2 (64-bit, Nov 15 2022)")
Info("  Models:  All")
Info("  Classes: All")

FunctionalBlocks{
     1 design {
           1  sbst 0ps {F:17992 ND:4066 DD:8890 PD:42 NC:1568 NO:3364 NS:62}
    }
}

FaultInfo{
    FubNum;
    TestNum;
    Attempts;
    NumPot;
}

FaultList{

Total             ND             DD           PD             NC             NO           NS Scope
----- -------------- -------------- ------------ -------------- -------------- ------------ -----
17992 4066  (22.60%) 8890  (49.41%) 42   (0.23%) 1568   (8.71%) 3364  (18.70%) 62   (0.34%) cv32e40p_top
17992 4066  (22.60%) 8890  (49.41%) 42   (0.23%) 1568   (8.71%) 3364  (18.70%) 62   (0.34%) -core_i
17992 4066  (22.60%) 8890  (49.41%) 42   (0.23%) 1568   (8.71%) 3364  (18.70%) 62   (0.34%) --ex_stage_i
17992 4066  (22.60%) 8890  (49.41%) 42   (0.23%) 1568   (8.71%) 3364  (18.70%) 62   (0.34%) ---alu_i (1363 hidden instances, 11248 faults)
 5972 1128  (18.89%) 3090  (51.74%) 42   (0.70%)  659  (11.03%)  997  (16.69%) 56   (0.94%) ----alu_div_i (775 hidden instances, 5972 faults)
  340  337  (99.12%)    0   (0.00%)  0   (0.00%)    0   (0.00%)    3   (0.88%)  0   (0.00%) ----popcnt_i (39 hidden instances, 340 faults)
  432   44  (10.19%)  156  (36.11%)  0   (0.00%)  114  (26.39%)  118  (27.31%)  0   (0.00%) ----ff_one_i (63 hidden instances, 432 faults)
}
#------------------------------------------------------------------------------
# Faults listed:       Prime: 0        Total: 0
#------------------------------------------------------------------------------
# Fault Coverage Summary
#
#                                        Prime             Total
#------------------------------------------------------------------------------
# Total Faults:                          17992             17992
#
# Dropped Detected                 DD     8890  49.41%      8890  49.41%
# Dropped Potential                PD       42   0.23%        42   0.23%
# Not Detected                     ND     4066  22.60%      4066  22.60%
# Not Strobed                      NS       62   0.34%        62   0.34%
# Not Observed                     NO     3364  18.70%      3364  18.70%
# Not Controlled                   NC     1568   8.71%      1568   8.71%
#
# Faults not detected yet:
#   Not simulated yet:                    4994  27.76%      4994  27.76%
#   Simulated 1 to 5 times:               4108  22.83%      4108  22.83%
#   Simulated 6 to 10 times:                 0   0.00%         0   0.00%
#   Simulated > 10 times:                    0   0.00%         0   0.00%
#
# Faults potentially detected:
#   Possible 1 to 5 times:                  42   0.23%        42   0.23%
#   Possible 6 to 10 times:                  0   0.00%         0   0.00%
#   Possible > 10 times:                     0   0.00%         0   0.00%
#
# Test Coverage                                 49.53%            49.53%
# Fault Coverage                                49.53%            49.53%
#------------------------------------------------------------------------------
